src/System.Net.Sockets/src/System/Net/Sockets/SocketInformation.cs

```diff
@@ -0,0 +1,131 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace System.Net.Sockets
+{
+    //[Serializable]
```

Comment:
I tried this, but simply updating the System.Runtime package references doesn't seem to be enough.  I'll file a separate issue for this.